source/docs/software/advanced-gradlerio/index.rst

```diff
@@ -11,3 +11,4 @@ GradleRIO is the mechanism that powers the deployment of robot code to the roboR
    code-formatting
    gradlew-tasks
    deploy-git-data
+   profiling-with-visualvm
```

Comment:
I'm not sure that advanded-gradlerio is the place for this. It does require modifying build.gradle. There isn't a great place that already exists, maybe telemetry?

Reply:
I'd rather not place it in telemetry, as I don't think we typically *want* to recommend users modify their `build.gradle`. If this is something common, we could enable it by default?

Reply:
I agree with @Daltz333 that users should only be enabling this for diagnosis and shouldn't be in the `build.gradle` deployed for matches. I think that Software Tools might not be a bad place for this, but not Telemetry for the same reason as Daltz said.

Reply:
I don't think I explained my concern with putting it in the advanced gradlerio section correctly. The other things in that section are permanent items that a team wants to change the build process. This is a temporary item to help with debugging. Because of that, I think it belongs elsewhere.

Reply:
Software Tools?